### DISCUSSION TITLE
Welcome to poap.js Discussions!

### DISCUSSION THREAD

**Discussion by rlajous** [poap-xyz/poap.js#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by coreyar** [comment]
Hi, I've picked up a few POAPs at different events and am a fan of the project. I'm interested in collaborating on building the SDK
